gulpfile.js

```diff
@@ -70,6 +72,11 @@ function handleError() {
   });
 }
 
+// cleanup ./dist in the initial build steps
+gulp.task('cleanup-dist', function (cb) {
+  del(['./dist/*'], cb);
+});
+
```

Comment:
Yeah, it might even mess things up if tasks are run in parallel and the cleanup task is run at the same time as the building task--eek.